pennylane/ops/qubit/observables.py

```diff
@@ -0,0 +1,221 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the available built-in discrete-variable
+quantum operations supported by PennyLane, as well as their conventions.
```

Comment:
I wonder if it makes sense to put these together with the other observables that are in the `parametrized_ops` file. Maybe these could be moved over there instead. I get the feeling that many users might look here for e.g. the pauli operators. 🤔